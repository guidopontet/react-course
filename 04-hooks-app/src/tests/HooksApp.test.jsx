import { shallow } from "enzyme";

import HooksApp from "../components/HooksApp";

describe('HooksApp', () => {
  test('should render', () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
})
