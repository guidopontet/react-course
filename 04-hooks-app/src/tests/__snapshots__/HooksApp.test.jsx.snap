// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HooksApp should render 1`] = `
<h1>
  HooksApp
</h1>
`;
